Bottom: f80ea18f2cb8578ebfc7a6f0cdec0e9008347712
Top:    f3bfe9259def378dfdf63838c7b605ce77190249
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:48:04 -0500

Refresh of all-normalize-vars

---
diff --git a/roles/docker-exim4/tasks/restore.yml b/roles/docker-exim4/tasks/restore.yml
index e863805..6910243 100644
--- a/roles/docker-exim4/tasks/restore.yml
+++ b/roles/docker-exim4/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: exists - exim4 archive
   stat:
-    path: '{{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }}'
+    path: '{{ exim4_docker_restore_dir }}{{ exim4_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_exim4_backup_file
@@ -22,7 +22,7 @@
 
 # - name: Set permissions for data-volume data
 #   file:
-#     path: '{{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }}'
+#     path: '{{ exim4_docker_restore_dir }}{{ exim4_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -30,7 +30,7 @@
 
 - name: exists - exim4 archive
   stat:
-    path: '{{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }}'
+    path: '{{ exim4_docker_restore_dir }}{{ exim4_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_exim4_backup_file
@@ -39,7 +39,7 @@
 # restore the archive
 
 - name: restore archive
-  shell: 'cat {{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }} | docker run -i --rm --volumes-from {{ exim4_dv_name }} {{ exim4_image_name }}:{{ docker_image_tag }} restore'
+  shell: 'cat {{ exim4_docker_restore_dir }}{{ exim4_backup_file }} | docker run -i --rm --volume {{ exim4_cv_name }}:/etc/exim4:z {{ exim4_image_name }}:{{ docker_image_tag }} restore'
   when: st_exim4_backup_file.stat.exists == True
 
 # *****************************************************************************
@@ -47,7 +47,7 @@
 
 - name: remove - exim4 archive
   file:
-    path: '{{ host_exim4_docker_restore_dir }}'
+    path: '{{ exim4_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index ac5ffcc..3a43141 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -11,7 +11,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-gitlab/tasks/restore.yml b/roles/docker-gitlab/tasks/restore.yml
index 90c7934..c95eb56 100644
--- a/roles/docker-gitlab/tasks/restore.yml
+++ b/roles/docker-gitlab/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
   register: gitlab_backup
 
 - name: data from tape
@@ -20,12 +20,12 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
   register: gitlab_backup
 
 # - name: File permissions
 #   file:
-#     path: '{{ host_gitlab_docker_restore_dir }}/{{ item }}'
+#     path: '{{ gitlab_docker_restore_dir }}/{{ item }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -46,7 +46,7 @@
 
 - name: Remove files
   file:
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 0c2f902..e25c4a7 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -12,7 +12,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_mantisbt_docker_restore_dir }}'
+    path: '{{ mantisbt_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
@@ -176,7 +176,7 @@
 
 - name: cleanup - import mysql config
   file:
-    path: '{{ host_mantisbt_docker_restore_dir }}/{{ item }}'
+    path: '{{ mantisbt_docker_restore_dir }}/{{ item }}'
     state: absent
   with_items:
   - largefile.cnf
diff --git a/roles/docker-mantisbt/tasks/restore.yml b/roles/docker-mantisbt/tasks/restore.yml
index 323c254..78f66b9 100644
--- a/roles/docker-mantisbt/tasks/restore.yml
+++ b/roles/docker-mantisbt/tasks/restore.yml
@@ -6,21 +6,21 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_static_archive
 
 - name: exists - SQL (files table)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_files_table_SQL_file
 
 - name: exists - SQL (remaining tables)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_SQL_file
@@ -36,7 +36,7 @@
 
 # - name: permissions - Static
 #   file:
-#     path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
+#     path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -44,7 +44,7 @@
 
 # - name: permissions - SQL (files table)
 #   file:
-#     path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
+#     path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -52,7 +52,7 @@
 
 # - name: permissions - SQL (remaining tables)
 #   file:
-#     path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
+#     path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -60,21 +60,21 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_static_archive
 
 - name: exists - SQL (files table)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_files_table_SQL_file
 
 - name: exists - SQL (remaining tables)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_SQL_file
@@ -93,7 +93,7 @@
 
 - name: Remove - Static and SQL
   file:
-    path: '{{ host_mantisbt_docker_restore_dir }}'
+    path: '{{ mantisbt_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 582a1fa..e16bb78 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -12,7 +12,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_wiki_docker_restore_dir }}'
+    path: '{{ wiki_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-mediawiki/tasks/restore.yml b/roles/docker-mediawiki/tasks/restore.yml
index 90b9b94..f2b3409 100644
--- a/roles/docker-mediawiki/tasks/restore.yml
+++ b/roles/docker-mediawiki/tasks/restore.yml
@@ -6,14 +6,14 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_static_archive
 
 - name: exists - SQL
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_SQL_file
@@ -29,7 +29,7 @@
 
 # - name: permissions - Static
 #   file:
-#     path: '{{ host_wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
+#     path: '{{ wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -37,7 +37,7 @@
 
 # - name: permissions - SQL
 #   file:
-#     path: '{{ host_wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
+#     path: '{{ wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -45,14 +45,14 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_static_archive
 
 - name: exists - SQL
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_SQL_file
@@ -71,7 +71,7 @@
 
 - name: Remove - Static and SQL
   file:
-    path: '{{ host_wiki_docker_restore_dir }}'
+    path: '{{ wiki_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-openldap/defaults/main.yml b/roles/docker-openldap/defaults/main.yml
index 8408262..019799f 100644
--- a/roles/docker-openldap/defaults/main.yml
+++ b/roles/docker-openldap/defaults/main.yml
@@ -34,8 +34,8 @@ openldap_image_name: '{{ docker_registry_username }}/ldap'
 openldap_container_name: '{{ docker_name_prefix }}ldap'
 
 # the name of the data-volume used by the container
-openldap_cv_name: '{{ docker_name_prefix }}ldap_CV'
-openldap_dv_name: '{{ docker_name_prefix }}ldap_DV'
+openldap_cv_name: '{{ docker_name_prefix }}openldap_CV'
+openldap_dv_name: '{{ docker_name_prefix }}openldap_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 openldap_docker_backup_dir: '{{ docker_backup_dir }}/LDAP'
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 45c83c2..cda73dc 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -5,8 +5,8 @@
 # A manuel step of setting the passwords into the LDAP database is required
 - assert:
     that:
-    - ldap_admin_password is defined
-    - ldap_proxyagent_password is defined
+    - openldap_admin_password is defined
+    - openldap_proxyagent_password is defined
 
 # *****************************************************************************
 # Setup the directory where the backup and restore is to take place
@@ -14,7 +14,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_openldap_docker_restore_dir }}'
+    path: '{{ openldap_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-openldap/tasks/restore.yml b/roles/docker-openldap/tasks/restore.yml
index 34c83e2..00f6ef9 100644
--- a/roles/docker-openldap/tasks/restore.yml
+++ b/roles/docker-openldap/tasks/restore.yml
@@ -6,8 +6,8 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_openldap_docker_restore_dir }}'
-  register: ldap_backup
+    path: '{{ openldap_docker_restore_dir }}'
+  register: openldap_backup
 
 - name: Get data-volume data from tape
   bacula:
@@ -16,7 +16,7 @@
     fileset: '{{ bacula_fileset }}'
     dest: '{{ bacula_dest }}'
     path_to_restore: '{{ openldap_docker_backup_dir }}'
-  when: ldap_backup.file_list == []
+  when: openldap_backup.file_list == []
 
 # - name: Set permissions for data-volume data
 #   file:
@@ -45,7 +45,7 @@
 
 - name: Remove database backup file
   file:
-    path: '{{ host_openldap_docker_restore_dir }}'
+    path: '{{ openldap_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-openssl/tasks/main.yml b/roles/docker-openssl/tasks/main.yml
index a3828f6..8f0eaf8 100644
--- a/roles/docker-openssl/tasks/main.yml
+++ b/roles/docker-openssl/tasks/main.yml
@@ -7,7 +7,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_openssl_docker_restore_dir }}'
+    path: '{{ openssl_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-openssl/tasks/restore.yml b/roles/docker-openssl/tasks/restore.yml
index 7740ffa..1cfc3d4 100644
--- a/roles/docker-openssl/tasks/restore.yml
+++ b/roles/docker-openssl/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: exists - openssl archive
   stat:
-    path: '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}'
+    path: '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_openssl_backup_file
@@ -22,7 +22,7 @@
 
 # - name: Set permissions for data-volume data
 #   file:
-#     path: '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}'
+#     path: '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -30,7 +30,7 @@
 
 - name: exists - openssl archive
   stat:
-    path: '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}'
+    path: '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_openssl_backup_file
@@ -44,9 +44,9 @@
     image: '{{ openssl_image_name }}:{{ docker_image_tag }}'
     name: openssl_restore
     volumes:
-      - '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}:{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}:z'
+      - '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}:{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}:z'
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
-    command: ['extract', '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}']
+    command: ['extract', '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}']
 
 - name: extract archive
   docker_container:
@@ -59,7 +59,7 @@
 
 - name: remove - openssl archive
   file:
-    path: '{{ host_openssl_docker_restore_dir }}'
+    path: '{{ openssl_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 40661b6..e39e3f9 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -7,7 +7,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
@@ -90,7 +90,7 @@
       - '{{ svn_dv_name }}:/var/lib/svn:z'
     env:
       SVN_HOSTNAME: '{{ container_addr_map.svn.hostname }}'
-      SVN_LDAP_PASSWORD: '{{ ldap_proxyagent_password }}'
+      SVN_LDAP_PASSWORD: '{{ openldap_proxyagent_password }}'
     image: '{{ svn_image_name }}:{{ docker_image_tag }}'
 
 # *****************************************************************************
